docs/usage.md

```diff
@@ -257,25 +120,17 @@ Attributes:
 
 You can see that the dataset contains a mixture of virtual variables backed by `ManifestArray` objects (`lat` and `lon`), and loadable variables backed by (lazy) numpy arrays (`air` and `time`).
 
-Loading variables can be useful in a few scenarios:
-
-1. You need to look at the actual values of a multi-dimensional variable in order to decide what to do next,
```

Comment:
Let's either keep this here or link to it from the  usage page.

Reply:
👍 I added a link in [`f450428` (#563)](https://github.com/zarr-developers/VirtualiZarr/pull/563/commits/f450428c11c130ab76d00db1ea4a3b80a58afa35)